### ISSUE TITLE
send authentication headers when calling aptly API

### ISSUE THREAD

**Issue by erdeerdeerde** [sdumetz/aptly-web-ui#6]
Ive noticed that when the UI calls eg. /api/repos the authentication header is not sent, thus i can not access the API.

**Comment by sdumetz** [comment]
Yep, looks like I forgot the  `{credentials: 'same-origin'}` option in fetch() calls. Not using Auth myself, thanks for reporting.

**Comment by benelgiac** [comment]
Encountering the same error, API is not usable with authentication. Any plans to fix this? :) thanks

**Comment by benelgiac** [comment]
By the way, I have *never* touched a line of js code, but based on your hint, the attached stupid patch made the trick for me and it now everything seems working with authentication.

[stupid_patch.diff.txt](https://github.com/sdumetz/aptly-web-ui/files/2166432/stupid_patch.diff.txt)
